docs/user-guide/config-management-plugins.md

```diff
@@ -20,47 +20,15 @@ The following sections will describe how to create, install, and use plugins. Ch
 
 There are two ways to install a Config Management Plugin:
 
-1. Add the plugin config to the Argo CD ConfigMap (**this method is deprecated and will be removed in a future 
+1. Add the plugin as a sidecar to the repo-server Pod.
+   This is a good option for a more complex plugin that would clutter the Argo CD ConfigMap. A copy of the repository is
+   sent to the sidecar container as a tarball and processed individually per application, which makes it a good option
+   for [concurrent processing of monorepos](../operator-manual/high_availability.md#enable-concurrent-processing).
+2. Add the plugin config to the Argo CD ConfigMap (**this method is deprecated and will be removed in a future 
    version**). The repo-server container will run your plugin's commands. This is a good option for a simple plugin that
    requires only a few lines of code that fit nicely in the Argo CD ConfigMap.
```

Comment:
Don't rely on numbers, name the bullets and then use the same names for the section titles below.

Use `a plugin` not `the plugin` as users can have multiple (I have two) -- and it's quite possible for them to migrate incrementally (I'd recommend that...).

```suggestion
* Sidecar plugin
  Add a plugin as a sidecar to the repo-server Pod.
   This is recommended. A copy of the repository is
   sent to the sidecar container as a tarball and processed individually per application, which makes it a good option
   for [concurrent processing of monorepos](../operator-manual/high_availability.md#enable-concurrent-processing).
* ConfigMap plugin
  (Deprecated)
  Add a plugin config to the Argo CD ConfigMap (**this method is deprecated and will be removed in a future 
   version**). The repo-server container will run your plugin's commands. This is a good option for a simple plugin that
   requires only a few lines of code that fit nicely in the Argo CD ConfigMap.
   
```